Package.swift

```diff
@@ -3,7 +3,7 @@
 import PackageDescription
 import Foundation
 
-let versionStr = "10.2.0"
+let versionStr = "10.2.1"
```

Comment:
This should be 10.3 as we have an entry in Enhancements.

Reply:
I thought the rule was actually that we bump the minor version if there is an entry in "Breaking changes"?

Reply:
No. 
Major.feature.bugfix

Major is a bit more up for negotiation....